compiler/rustc_target/src/asm/mod.rs

```diff
@@ -177,6 +179,7 @@ pub enum InlineAsmArch {
     Hexagon,
     Mips,
     Mips64,
+    Spirv,
```

Comment:
Small nit: this should be `SpirV`.

Reply:
The formal capitalization is `SPIR-V`, all caps, so just checking, should it be `SPIRV` or `SpirV`? Looks like `RISC-V` is also all caps and that's spelled `RiscV` here, so yeah, `SpirV`?